frontend/src/main/message-handlers.ts

```diff
@@ -43,6 +44,10 @@ const messageHandlers: MessageHandlers = {
     await openFolder(asset_id);
     return { ok: true };
   },
+  'assets:open-Maya': async (_, { asset_id}) => {
```

Comment:
should probably change casing to "open-maya" to match other call names but not super important